toolkit/mainchain-follower/db-sync-follower/src/db_model.rs

```diff
@@ -310,6 +310,37 @@ pub(crate) struct BlockTokenAmount {
 	pub amount: NativeTokenAmount,
 }
 
+#[derive(Debug, Clone, sqlx::FromRow, PartialEq)]
+pub(crate) struct StakePoolDelegationOutputRow {
+	pub epoch_stake_amount: StakeDelegation,
+	pub pool_hash_raw: [u8; 28],
+	pub stake_address_hash_raw: [u8; 29],
+	pub stake_address_script_hash: Option<[u8; 28]>,
+}
+
+pub(crate) async fn get_stake_pool_delegations(
+	pool: &Pool<Postgres>,
+	epoch: EpochNumber,
+) -> Result<Vec<StakePoolDelegationOutputRow>, SqlxError> {
+	Ok(sqlx::query_as::<_, StakePoolDelegationOutputRow>(
+		"
+SELECT
+	epoch_stake.amount AS epoch_stake_amount,
+	pool_hash.hash_raw AS pool_hash_raw,
+	stake_address.hash_raw AS stake_address_hash_raw,
+	stake_address.script_hash AS stake_address_script_hash
+FROM
+			   epoch_stake
+	INNER JOIN stake_address      ON epoch_stake.addr_id = stake_address.id
+	INNER JOIN pool_hash          ON epoch_stake.pool_id = pool_hash.id
+WHERE epoch_stake.epoch_no = $1
```

Comment:
It's only 2% of entries, but `AND epoch_stake.amount > 0` also doesn't add too complexity to the code.